content/docs/esc/integrations/infrastructure/pulumi-iac/_index.md

```diff
@@ -12,16 +12,20 @@ menu:
 
 With support for Pulumi ESC built into the Pulumi CLI, you can expose an environment's settings and secrets to any or all of your Pulumi stacks, bypassing the need to define and maintain individual configuration settings or secrets "locally" in Pulumi config files. The optional `pulumiConfig` key enables this.
 
-The following example updates the `myorg/myapp-dev` environment by adding a `pulumiConfig` block. This block specifies the [Pulumi configuration](/docs/concepts/config/) settings to expose to the Pulumi stack at runtime:
+{{% notes type="tip" %}}
```

Comment:
Curious why we use type=tip here? I've not seen tip used before. We usually do type=info

Reply:
No particular reason, I can change it to info.